sql/core/src/test/scala/org/apache/spark/sql/GeneratorFunctionSuite.scala

```diff
@@ -357,6 +357,30 @@ class GeneratorFunctionSuite extends QueryTest with SharedSparkSession {
     val df = Seq(1, 2, 3).toDF("v")
     checkAnswer(df.select(explode(array(min($"v"), max($"v")))), Row(1) :: Row(3) :: Nil)
   }
+
+  test("SPARK-37947: lateral view <func>_outer()") {
```

Comment:
before this PR, `<func>_outer()` can only be used in SELECT?

Reply:
>before this PR, <func>_outer() can only be used in SELECT?

Correct